internal/exec/execabs.go

```diff
@@ -0,0 +1,60 @@
+package exec
+
+import (
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"unsafe"
+)
+
+func relError(file, path string) error {
+	return fmt.Errorf("%s resolves to executable in current directory (.%c%s)", file, filepath.Separator, path)
+}
+
+// LookPath searches for an executable named file in the directories
+// named by the PATH environment variable. If file contains a slash,
+// it is tried directly and the PATH is not consulted. The result will be
+// an absolute path.
+//
+// LookPath differs from exec.LookPath in its handling of PATH lookups,
+// which are used for file names without slashes. If exec.LookPath's
+// PATH lookup would have returned an executable from the current directory,
+// LookPath instead returns an error.
+func LookPath(file string) (string, error) {
```

Comment:
I don't understand this. Can you explain what are we trying to do here and why?

Reply:
This article will explain it much better than I and the reason for it: https://blog.golang.org/path-security

This is from the mitigation for the issue in that article, not something I'd written. We already resolve the path to the init process for job containers ourselves before we'd ever get to calling LookPath to do it also (it's called during Run/Start etc. on `exec.Cmd`) so this isn't too much of an issue, but if we ever re-use the forked exec package for something else it will be nice to have in.

Reply:
So should this `execabs.go` file be a part of the first commit where you just bring in the forked copy of os/exec?

Reply:
I didn't just because it's not actually apart of the stdlib. Would a comment at the top specifying where it's from and the reason suffice?